core/src/main/resources/jenkins/model/GlobalUIConfiguration/config.groovy

```diff
@@ -0,0 +1,7 @@
+package jenkins.model.GlobalUIConfiguration
```

Comment:
Any reason for having this template groovy and not jelly?

Reply:
It's the typical approach. See all the configurations in https://github.com/jenkinsci/jenkins/tree/master/core/src/main/resources/jenkins/model. For example, [GlobalPluginConfiguration](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/jenkins/model/GlobalPluginConfiguration/config.groovy)